inframon/tools/enclosure_api.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+
+""" This module implements tool for Enclosure APIs.
+    Any communication to storage hardware happens via this code while using enclosure APIs.
+"""
+
+
+from abc import ABC, abstractmethod
+
+import hashlib
+import json
+import threading
+from typing import Any, List
+
+import requests
+from urllib3.exceptions import InsecureRequestWarning
+
+from inframon.comps import StorageComponents
+from inframon.tools.tool import Tool, ToolResponse
+#from inframon.config import InfraMonConfig
```

Comment:
if not useful can we remove the line?

Reply:
There is commented code in the class. That code with this import should be enabled once InfraMonConfig is implemented.